projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-saving-post/Readme.md

```diff
@@ -0,0 +1,27 @@
+# usePostJustSaved() hook
+React hook to detect when the post is just saved.
+Also, it accepts a dependency array passed to useEffect hook.
+
+```es6
+import { usePostJustSaved } from '../../hooks/use-saving-post';
+
+function SavingPostLabel() {
+	usePostJustSaved( function() {
+		console.log( 'The post has been saved!' );
+	} );
+}
+```
+
+# usePostJustPublished() hook
+React hook to detect when the post is just saved.
```

Comment:
Should this be `is just published`?

Reply:
it is!!! 🤦 Going to fix it in a follow-up PR. Thanks

Reply:
actually fixed here 291a49aa451f7fe24cf70e8052a6bb66e1a81b79. Since I had to rebase with master.